scripts/tests/test_zkai_native_seq32_attention_mlp_deterministic_adjacent_label_policy_gate.py

```
@@ -0,0 +1,281 @@
+            self.assertIn(
+                "adjacent_label_probe_a\tsupported_label",
+                tsv_out.read_text(encoding="utf-8"),
+            )
+
+    def test_write_outputs_rejects_invalid_payload(self) -> None:
+        payload = copy.deepcopy(self.__class__.payload)
+        payload["policy_summary"]["worst_supported_saving_typed_bytes"] = 0
+        payload["payload_commitment"] = self.gate.payload_commitment(payload)
+        with tempfile.TemporaryDirectory(dir=self.gate.EVIDENCE_DIR) as tmpdir:
+            tmp = pathlib.Path(tmpdir)
+            with self.assertRaisesRegex(
```

Ruff (0.15.13)

[warning] Use `pytest.raises` instead of unittest-style `assertRaisesRegex`

Replace `assertRaisesRegex` with `pytest.raises`

(PT027)